### ISSUE TITLE
[Request/suggestion] Exposing delay variable for user input

### ISSUE THREAD

**Issue by Shrimpey** [TomGrobbe/MenuAPI#16]
**[Request/suggestion]**
Hi, it would be nice to have delay variable (used for faster left/right inputs in menu when holding down button) exposed for user to tune. I'm talking about line _484_ and _512_ of MenuController.cs:
`if (times > 2)
                                    {
                                        delay = 150;
                                    }`
I was looking for a way to speed up left/right inputs for sliders/lists (with bigger lists it takes long time to scroll through them) and thought it would be nice to have some sort of public var to tune the delay.

**Comment by Local9** [comment]
I get what you're requesting and its not much work to add it in BUT wouldn't it be more user friendly to have larger lists in a vertical list rather than in a slider?

**Comment by Shrimpey** [comment]
I guess you're right, the feature I needed it for was really specific, I implemented a slider for RGB colors (so 256 values) and wanted to keep it in a slider/list to make it easier to tune (compared to sth like direct keyboard input). I ended up doing it with steps of 5 instead of 1 and adding keyboard input for fine tuning.

**Comment by Local9** [comment]
Ah RGB... I personally would of shot towards an NUI panel for that, but 700 time cycle entries.... I need to see this. (I am always interested in what people do)

**Comment by TomGrobbe** [comment]
Yeah I'll have to speed things up either way, because it's really slow currently I do agree. I'll add multiple phases of scrolling, the longer you hold it down, the faster it'll scroll (with a limit of 2 or 3 speed increments total).

**Comment by Shrimpey** [comment]
> Yeah I'll have to speed things up either way, because it's really slow currently I do agree. I'll add multiple phases of scrolling, the longer you hold it down, the faster it'll scroll (with a limit of 2 or 3 speed increments total).
> 
> No eta for this though.

Yup, that would be perfect.

**Comment by TomGrobbe** [comment]
Added in https://github.com/TomGrobbe/MenuAPI/commit/45f30260980896803bf2be8fd3f6ca4870bdff15, it'll be in the main release once RedM is ready for a full release.

**Comment by TomGrobbe** [comment]
Merged.
